server/httpsrv/httpsrv.go

```diff
@@ -63,6 +65,13 @@ type T struct {
 	errorCh    chan error
 }
 
+func init() {
+	var err error
+	if jsonContentTypePattern, err = regexp.Compile("^application/(?:.*\\+)?json$"); err != nil {
```

Comment:
what is the rationale for this regex. You could easily check with `strings.HasSuffix('json')` and that should cover.

* application/json
* text/json
* application/x-json
* text/x-json

Reply:
That was actually an external contribution by @solsson. It seems to a least inclusive change from `application/json`. I do not really care `regex` or `HasPrefix`

Reply:
Up to you. I didn't reflect over alternatives to using regex. If there's performance or maintenance advantages, please switch.